index.js

```diff
@@ -1,16 +1,21 @@
 'use strict';
 var spliceString = require('splice-string');
 
+var sets = {
+	'0': 0,	'1': 22, '2': 22, '3': 23, '4': 24, '7': 27, '8': 28, '9': 2, '30': 39, '31': 39, '32': 39, '33': 39, '34': 39, '35': 39, '36': 39, '37': 39, '90': 3, '40': 49, '41': 49, '42': 49, '43': 49, '44': 49, '45': 49, '46': 49, '47': 4
+};
```

Comment:
I could, however, I would need to disassemble it anyway into the above set - in looking at it, it only seems to expose them as strings, and I don't really need that.